dataeng/jobs/analytics/EventExportIncrementalLarge.groovy

```diff
@@ -0,0 +1,45 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.data_czar_keys_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.from_date_interval_parameter
+import static org.edx.jenkins.dsl.AnalyticsConstants.to_date_interval_parameter
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+class EventExportIncrementalLarge {
+    public static def job = { dslFactory, allVars ->
+        allVars.get('ENVIRONMENTS').each { environment, env_config ->
+            dslFactory.job("event-export-incremental-large-$environment") {
+                authorization {
+                    allVars.get('USER_ROLES').each { github_id, roles ->
+                        roles.each {
+                            permission(it, github_id)     // it is an implicit parameter corresponding to the current element
+                        }
+                    }
+                }
+                logRotator common_log_rotator(allVars, env_config)
+                parameters common_parameters(allVars, env_config)
+                parameters from_date_interval_parameter(allVars)
+                parameters to_date_interval_parameter(allVars)
+                parameters {
+                    stringParam('OUTPUT_ROOT', allVars.get('OUTPUT_ROOT'))
+                    stringParam('EXPORTER_CONFIG', allVars.get('EXPORTER_CONFIG'), 'Exporter configuration relative to analytics-secure/analytics-exporter')
+                    stringParam('ONLY_ORGS', allVars.get('ONLY_ORGS'), "i.e. --org-id [\"FooX\",\"BarX\"]")
+                    stringParam('DATA_CZAR_KEYS_BRANCH', allVars.get('DATA_CZAR_KEYS_BRANCH'), '')
+                }
+                multiscm common_multiscm(allVars) >> secure_scm(allVars) >> data_czar_keys_scm(allVars)
+
+                triggers common_triggers(allVars, env_config)
+                wrappers common_wrappers(allVars)
+                publishers common_publishers(allVars)
+                steps {
+                    shell(dslFactory.readFileFromWorkspace("dataeng/resources/event-export-incremental-large-${environment}.sh"))
```

Comment:
Do we need separate scripts, or can we just use the existing scripts.  And for that matter, is there a reason that we don't just have one single script for all four event export jobs?   Seems that they differ only in the --environment variable and the ONLY_ORGS.   Which should be blank for the other jobs, so would work fine there with a blank default.